practice-one/pages/components.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+  <body>
+    <button class="btn btn-secondary">Let's Talk</button>    
+    <button class="btn btn-primary">Subscribe Now</button>
+
+    <div class="card-item">
+      <figure class="card-header">
+        <img src="../assets/icons/icon-ux-interface.svg" alt="UX Design"/>
+        <p>01</p>
+      </figure>
+      <div class="card-body">
+        <h2 class="card-title card-title-large">UX Design</h2>
+        <h3 class="card-details">Mobile App, Website</h3>
+        <p class="card-description">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Saepe laborum sed commodi 
+          minus tempore repudiandae eos odio sapiente at incidunt, provident dicta! 
+          Incidunt temporibus iure impedit illum facilis sint reiciendis?</p>
+      </div>
+    </div>
+    <br>
+
+    <div class="card-item">
+      <div class="card-rates">
+        <button class="rate"></button>
+        <button class="rate"></button>
+        <button class="rate"></button>
+        <button class="rate"></button>
+        <button class="rate"></button>
+      </div>
+      <div class="card-body">
+        <p class="card-description">
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Unde sequi ab 
+          illum ea vero sint molestias expedita id at veritatis adipisci voluptatem 
+          reiciendis voluptatibus explicabo quis, assumenda iste rerum obcaecati.</p>
+      </div>
+      <div class="card-footer">
+        <figure class="card-author">
+          <img src="../assets/images/testimonial-woman-brown.png" alt="Woman with brown"/>
+        </figure>
+        <div class="card-info">
+          <p>Philip Levin</p>
+          <p>CEO/ Creative IT</p>
+        </div>
+        <figure class="card-icons">
+          <img src="../assets/icons/icon-double-quotes.svg" alt="vector"/>
+        </figure>
+      </div>
+    </div>
+    <br>
+
+    <div class="card-item">
+      <figure class="card-header card-img">
+        <img src="../assets/images/blog-graphic-design.png" alt="Graphic Design"/>
+      </figure>
+      <div class="card-body">
+        <h2 class="card-title">35 Stellar Graphic Design Blogs to Keep YouEducated and Inspired</h2>
+        <div class="card-details">
+          <div class="details">
```

Comment:
Do we use this class anywhere? If not, can we define class based on the content?